ext/standard/php_http.h

```diff
@@ -25,6 +25,5 @@ PHPAPI void php_url_encode_hash_ex(HashTable *ht, smart_str *formstr,
 				const char *key_prefix, size_t key_prefix_len,
 				const char *key_suffix, size_t key_suffix_len,
 				zval *type, const char *arg_sep, int enc_type);
-#define php_url_encode_hash(ht, formstr)	php_url_encode_hash_ex((ht), (formstr), NULL, 0, NULL, 0, NULL, 0, NULL)
```

Comment:
How is this related to https://github.com/php/php-src/blob/e547ea43c1a490d445b3d58294d12bdb17ffdf60/ext/standard/http.c#L23? If related at all, of course